config/xml_schemas/expected_fails_file.xsd

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+
+  <!-- simple elements -->
+  <xs:element name="category" type="xs:string"/>
+  <xs:element name="status" type="xs:string"/>
+  <xs:element name="issue" type="xs:string"/>
+  <xs:element name="comment" type="xs:string"/>
+
+  <!-- complex elements -->
+
+  <xs:element name="expectedFails">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="unbounded" ref="test"/>
+      </xs:sequence>
+      <xs:attribute name="version" use="required" type="xs:decimal"/>
+    </xs:complexType>
+  </xs:element>
+
+  <xs:element name="test">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="1" ref="category"/>
+        <xs:element minOccurs="1" maxOccurs="unbounded" ref="phase"/>
+      </xs:sequence>
+      <xs:attribute name="name" use="required" type="xs:NCName"/>
+    </xs:complexType>
+  </xs:element>
+
+  <xs:element name="phase">
```

Comment:
I would normally define an element (such as phase below) before I use it (as in test above) but this is purely a style issue.

Reply:
That makes sense, and I like the idea of that consistency. I have just pushed a commit where I made that change.